internal/providers/nscloud/api/rpc.go

```diff
@@ -34,22 +34,11 @@ type API struct {
 
 var Endpoint API
 
-func fetchTenantToken(ctx context.Context) (string, error) {
-	token, err := auth.LoadTenantToken()
-	if err != nil {
-		if os.IsNotExist(err) {
-			// Pass user auth for now.
-			// TODO exchange user auth for tenant tokens.
-			return auth.GenerateToken(ctx)
-		}
-
-		return "", err
+func MakeAPI(endpoint string) API {
+	fetchTenantToken := func(ctx context.Context) (string, error) {
```

Comment:
Let's inline this.

Reply:
You mean use `FetchToken: ExchangeToken` the `fnapi.Call` objects? It doesn't work due to type (`ExchangeToken` has optional `scopes` parameter).

Reply:
🙈 yep - true - thanks!